content/en/docs/components/model-registry/getting-started.md

```diff
@@ -0,0 +1,170 @@
++++
+title = "Getting started"
+description = "Getting started with Model Registry using examples"
+weight = 30
+
++++
+
+This guide shows how to get started with Model Registry and run a few examples using the
+command line or Python clients.
+
+At this time, there is not yet an UI for Model Registry; therefore, this documentation focus on backend services and APIs.
```

Comment:
this sentence might be rewritten i.e. At this time, the Model Registry does not include a web-based User Interface (UI), therefore this documentation focuses on backend services and APIs.

Reply:
implemented with https://github.com/kubeflow/website/pull/3698/commits/84f3859cb79d071fc535a4ae2350517995ad4ed6 , thanks!